pkg/cableengine/healthchecker/healthchecker.go

```diff
@@ -123,27 +128,36 @@ func (h *controller) endpointCreatedOrUpdated(obj runtime.Object, _ int) bool {
 		return false
 	}
 
-	if endpointCreated.Spec.GetHealthCheckIP(k8snet.IPv4) == "" || endpointCreated.Spec.CableName == "" {
-		logger.Infof("HealthCheckIP (%q) and/or CableName (%q) for Endpoint %q empty - will not monitor endpoint health",
-			endpointCreated.Spec.GetHealthCheckIP(k8snet.IPv4), endpointCreated.Spec.CableName, endpointCreated.Name)
-		return false
-	}
-
 	h.Lock()
 	defer h.Unlock()
 
-	if pingerObject, found := h.pingers[endpointCreated.Spec.CableName]; found {
-		if pingerObject.GetIP() == endpointCreated.Spec.GetHealthCheckIP(k8snet.IPv4) {
-			return false
+	for _, family := range h.config.SupportedIPFamilies {
+		healthCheckIP := endpointCreated.Spec.GetHealthCheckIP(family)
+		if healthCheckIP == "" {
```

Comment:
I can see that current code also checks if endpointCreated.Spec.CableName == "" , do we need to keep it ?